command/server/tls_util.go

```diff
@@ -0,0 +1,161 @@
+package server
```

Comment:
Is there a better tls-specific package to place these util in? If they are only going to be used by the CLI, feel free to ignore this :)

Reply:
These are just for the CLI and we would probably not use them elsewhere.